src/engine.cpp

```diff
@@ -18,13 +18,9 @@ std::vector<std::string> Util::getFilesInDirectory(const std::string &dirPath) {
 }
 
 void Logger::log(Severity severity, const char *msg) noexcept {
-    // Would advise using a proper logging utility such as
-    // https://github.com/gabime/spdlog For the sake of this tutorial, will just
-    // log to the console.
-
     // Only log Warnings or more important.
     if (severity <= Severity::kWARNING) {
-        std::cout << msg << std::endl;
+        spdlog::warn(msg);
     }
```

Comment:
Can you also add the other cases please. 

So something like:
```
 if (severity == Severity::kWARNING) {
        spdlog::warn(msg);
 } else if (severity == Severity::kINFO) { 
       // log to spdlog info... 
```

That way the TensorRT log level is controlled by the environment variable too.

Reply:
Makes sense. Let me update it.